platform/lang-impl/src/com/intellij/ide/util/AbstractTreeClassChooserDialog.java

```diff
@@ -55,6 +56,7 @@ public abstract class AbstractTreeClassChooserDialog<T extends PsiNamedElement>
   private final @NotNull Project myProject;
   private final GlobalSearchScope myScope;
   private final @NotNull Filter<T> myClassFilter;
+  private final @Nullable Comparator<? super NodeDescriptor<?>> myComparator;
```

Comment:
Looks like this should be `@NotNull` instead because you set it to `AlphaComparator.INSTANCE` if `null` is passed into the constructor.